tools/pot/openvino/tools/pot/graph/node_utils.py

```diff
@@ -191,7 +191,12 @@ def get_quantized_input_key(quantized_node):
     If input node of quantized node have one output port -> key is name of fq_input node.
     Otherwise, key is tuple (fq_input name, output port number)
     """
-    quantized_input = get_node_input(quantized_node, 0)
+    if quantized_node.type == 'Add':
```

Comment:
It looks like we should have a test for a couple of cases.

Reply:
Added. Review it again, please.